File: TDD Boot Camp/Part 01/Code/OOPrinciples/src/com/oop/abstraction/Reactangle.java
package com.oop.abstraction;

public class Reactangle {
	private Size size;
	private Point point;
	Reactangle(Point point, Size size){
		this.point = point;
		this.size = size;
	}
	public Point getPoint() {
		return point;
	}
	public Size getSize() {
		return size;
	}
}

File: TDD Boot Camp/Part 01/Code/C#/OOPrinciples/Abstraction/Reactangle.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Abstraction
{
    public class Reactangle
    {
        public Size Size { get; set; }
        public Point Point { get; set; }
        public Reactangle(Point _point, Size _size)
        {
            Point = _point;
            Size = _size;
        }
    }
}
